### DISCUSSION TITLE
Are meshcentral node details available offline?

### DISCUSSION THREAD

**Discussion by DaanSelen** [Ylianst/MeshCentral#6423] [General]
Hello all, I am programming my own extension for MeshCentral in Python. And the question, can I query a node its data such as sysinfo and networkinfo when its offline? If no, why not? I need to make a overview for a end-of-month report for the customer being managed by MeshCentral?

**Comment by si458** [comment]
Yes it's stored in the database so u can see its info when offline
You use the meshctrl.js and use the `deviceinfo` command to get infor about a device using its nodeid

**Reply by DaanSelen** [reply]
I will look inwards into my code, thanks for the clarification.

**Reply by DaanSelen** [reply]
It was my own code which caused it.
